extensions/interactions/ImageClickInput/directives/image-click-input-interaction.component.html

```diff
@@ -24,6 +24,17 @@
     width: 10px;
     z-index: 10;
   }
+  .oppia-select-image-region-cursor {
+    background: red;
```

Comment:
Prefer hex code for colours (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css).)

Reply:
Done. PTAL!